### PR TITLE
feat: add pay per result docs

### PR DESCRIPTION
**Pull request by mhamas**
Basic documentation on what pay per result actors are. Note this is only first version. I think the good approach here is to get out something basic as soon as possible and iterate later.

For better reading (images cannot be easily rendered locally)
CleanShot 2023-02-22 at 16 33 17@2x

### CODE REVIEW

**Comment by B4nan** [comment]
Keep in mind this repo still has old content, I will move it on monday when I am back from vacation (pr is here already). So you should probably move this to the apify-docs-v2 repo.

File: content/docs/actors/pay_per_result_actors.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+title: Actors paid per result
+description: Learn how actors that are paid per result work. Find out how this differ from the traditional "pay for usage" model and how this benefits you.
```
Comment (TheoVasilis) [inline content/docs/actors/pay_per_result_actors.md:3]: Find out how this model differs from the traditional "pay for usage" model and how this benefits you.

File: content/docs/actors/pay_per_result_actors.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+menuWeight: 7.4
+paths:
+# NOTE: IF ADDING A NEW PATH, LEAVE THE OLD ONES FOR REDIRECTS
+    - actors/pay-per-result-actors
+---
+
+This page explains a new pricing model for actors in the [Apify store](https://apify.com/store) that allows you to pay only for the results an actor generates and not the underlying platform usage. This makes it transparent and easy to estimate upfront costs. If you have any feedback or would like to ask something, please join our [Discord](https://discord.gg/qkMS6pU4cF) community and let us know!
```
Comment (TheoVasilis) [inline content/docs/actors/pay_per_result_actors.md:11]: This page explains how the new pricing model for actors in [Apify Store](https://apify.com/store) allows you to pay only for the results an actor generates and not the underlying platform usage.

File: content/docs/actors/pay_per_result_actors.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+# Traditional approach - paying for usage
+
+Traditionally, public actors in the [Apify Store](https://apify.com/store) come in two flavors: free and [paid actors](https://docs.apify.com/platform/actors/paid-actors). The latter requires a monthly fee to rent the actor and use it after its trial period. When you run either a free or paid actor from the store, they generate platform usage that is subtracted from your prepaid usage, which is either $5 on the free plan or prepaid usage given by your subscription plan.
```
Comment (TheoVasilis) [inline content/docs/actors/pay_per_result_actors.md:15]: There are two types of public actors in [Apify Store](https://apify.com/store): free and [paid](https://docs.apify.com/platform/actors/paid-actors).

File: content/docs/actors/pay_per_result_actors.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+![Free and paid actors in Apify Store]({{@asset actors/images/free_vs_paid_actors.webp}})
+
+This platform usage includes components such as [compute](https://docs.apify.com/platform/actors/running/compute-units), operations on [storages](https://docs.apify.com/platform/storage), and usage of [residential proxies](https://docs.apify.com/platform/proxy/residential-proxy) or [SERPs](<https://docs.apify.com/platform/proxy/google-serp-proxy>). You can see the usage your actor run generated on the run detail page.
+
+![Example run usage]({{@asset actors/images/example_run_usage.webp}})
+
+In this model, it is quite difficult to know beforehand exactly how much usage the actor will generate when you run it so you cannot easily estimate your costs upfront.
```
Comment (TheoVasilis) [inline content/docs/actors/pay_per_result_actors.md:23]: With this model, it is quite difficult to know beforehand exactly how much usage the actor will generate when you run it, so you cannot easily estimate your costs upfront.

File: content/docs/actors/pay_per_result_actors.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+## Paying only for results instead
+
+With **pay per result** model you will only pay for the results that an actor returns when you run it, and you will not be charged for the underlying platform usage. Technically, **results** you are charged for are the clean items in the default [dataset](https://docs.apify.com/platform/storage/dataset) created by the actor run.
```
Comment (TheoVasilis) [inline content/docs/actors/pay_per_result_actors.md:27]: With the **pay per result** model, you only pay for the results that an actor returns when you run it, and you are not charged for the underlying platform usage. Technically, **results** you are charged for are the clean items in the default [dataset](https://docs.apify.com/platform/storage/dataset) created by the actor run.

File: content/docs/actors/pay_per_result_actors.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+<!-- TODO - add a bit about the ability to set the maximum limits actor should return -->
+
+### How do I know an actor is paid per result?
+Actor that is paid per result will display the price per 1,000 items in the store.
```
Comment (TheoVasilis) [inline content/docs/actors/pay_per_result_actors.md:32]: An actor that is paid per result will display the price per 1,000 items in the store.

File: content/docs/actors/pay_per_result_actors.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+![Pay per result actor in store]({{@asset actors/images/pay_per_result_actor_store_card.webp}})
+
+When you try the actor on the platform, you will see that the actor is paid per result next to the actor name.
+
+![Actor paid per result in Console]({{@asset actors/images/console_pay_per_result_tag.webp}})
+
+### Do I need to pay monthly rental fee to run the actor?
```
Comment (TheoVasilis) [inline content/docs/actors/pay_per_result_actors.md:40]: ### Do I need to pay a monthly rental fee to run the actor?

File: content/docs/actors/pay_per_result_actors.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+No, the actor is free to run, you only pay for the results.
```
Comment (TheoVasilis) [inline content/docs/actors/pay_per_result_actors.md:41]: No, the actor is free to run. You only pay for the results.

File: content/docs/actors/pay_per_result_actors.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+
+### What happens when I interact with the dataset after the run finishes?
+Under the **pay per result** model, all platform costs generated **during the run of an actor** are not charged towards your account; you pay for the results instead. After the run finishes, any interactions with the default dataset storing the results, such as reading the results or writing additional data, will incur the standard platform usage costs. But do not worry, in the vast majority of cases, you only want to read the result from the dataset and that costs near to nothing.
+
+### Do I pay for the storage of results on the Apify platform?
+You will still be charged for the timed storage of the data in the same fashion as before. You can always decide to delete the dataset to reduce your costs after you exported the data from the platform. By default, any unnamed dataset will be automatically removed after your data retention period, so it most cases this is nothing to worry about.
```
Comment (TheoVasilis) [inline content/docs/actors/pay_per_result_actors.md:47]: You will still be charged for the timed storage of the data in the same fashion as before. You can always decide to delete the dataset to reduce your costs after you export the data from the platform. By default, any unnamed dataset will be automatically removed after your data retention period, so in most cases, this is nothing to worry about.

**Review by jancurn — APPROVED** [review]

**Comment by B4nan** [comment]
Let me say it one more time, this should be targetting the apify-docs-v2 repo or wait before #517 gets merged.

**Comment by mhamas** [comment]
@B4nan I don't need to deploy it until next week, so it should be fine. I looked at v2 repo as well, but I wasn't sure where it would be the best to put this content and how it exactly interacts with the old content in this repo. Let's discuss next week when you are back, it's not urgent. Thx

**Review by mtrunkat — APPROVED** [review]
Looks good. I plan to transform this section a lot but this will serve as a great base.

**Comment by mtrunkat** [comment]
Closing in favor of #532

**Pull request closed without merging**